clang/test/SemaCXX/warn-unused-private-field.cpp

```diff
@@ -20,6 +20,20 @@ class SpaceShipDefaultCompare {
   int operator<=>(const SpaceShipDefaultCompare &) const = default;
 };
 
+class EqDefaultCompareOutOfClass {
+  int used;
```

Comment:
A comment here pointing out that we're validating that `used` doesn't get diagnosed would be helpful.